includes/Plugin.php

```diff
@@ -170,7 +170,7 @@ class Plugin {
 	 */
 	public function register() {
 		// Plugin compatibility / polyfills.
-		add_action( 'wp', [ $this, 'load_amp_plugin_compat' ] );
+		add_action( 'init', [ $this, 'load_amp_plugin_compat' ] );
```

Comment:
@swissspidy Why was this hooked on `wp`? Would it effect anything to change this.

Reply:
I think because this was the earliest point where those could be loaded.

Counter question: why is it changed here?

Reply:
As mentioned in above comment, this is not needed here so reverting. This may come in some other subsequent PRs

Reply:
Please refer[ this function in functions.php](https://github.com/rtCamp/web-stories-wp/pull/189/files#diff-1c08b27bcf43583c75cbc442bc401638be3427586fe544f3389e22c9979fa1afR149) file.

Here, field_states is calling Renderer's init method which in turn checks for `is_amp_endpoint` function. This function is getting added on `wp` hook, `wp` hook would not get fired in admin, so it will throw fatal error.

@spacedmonkey I remember previous instance where you commented on PR that we should not check for 

```php
function_exists( 'is_amp_endpoint' )
```

because it will always be available, but this is one example where it can cause issues.

Reply:
There is a reason it this was in `wp` and I think this will make issues loading it here. I think it might have some of the issues we are seeing in CI.

Reply:
Continuing discussion at https://github.com/rtCamp/web-stories-wp/pull/201#discussion_r554991638

Reply:
Pe discussion at #201 we now have a new `is_amp()` function we can use everywhere.

Means we can undo this change here.